minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -631,7 +629,9 @@ public synchronized void start() throws IOException, InterruptedException {
         throw new IllegalStateException("No Compactor queues configured.");
       }
       for (String name : queues) {
-        control.startCompactors(Compactor.class, getConfig().getNumCompactors(), name);
+        // ELASTICITY_TODO need to reevalute numCompactors and maybe num scan servers. The following
+        // was calling numCompactors, but it was changed to 1.
```

Comment:
I agree with this comment. Mini needs to be modified such that Compactors and ScanServers are first class citizens.

Reply:
Fixed by #3533